scheduler/wmap/wmap.go

```diff
@@ -282,10 +285,11 @@ func (c *CollectWorkflowMapNode) AddConfigItem(ns, key string, value interface{}
 }
 
 type ProcessWorkflowMapNode struct {
-	Name         string                   `json:"plugin_name"yaml:"plugin_name"`
-	Version      int                      `json:"plugin_version"yaml:"plugin_version"`
-	ProcessNodes []ProcessWorkflowMapNode `json:"process,omitempty"yaml:"process"`
-	PublishNodes []PublishWorkflowMapNode `json:"publish,omitempty"yaml:"publish"`
+	// required: true
+	PluginName    string                   `json:"plugin_name"yaml:"plugin_name"`
+	PluginVersion int                      `json:"plugin_version"yaml:"plugin_version"`
```

Comment:
Changed `Name` to `PluginName` and 'Version` to `PluginVersion` for the consistency between variable names and JSON representation. The consequence is:
* `go-swagger` generated client can reference back easily and without patching shell scripts.
*  variable name changes are invisible to customers. Changes only affect tests inside code base.